docker/vars.local.sh

```diff
@@ -5,3 +5,5 @@ export GPC_CONSUMER_GPC_GET_URL="http://localhost:8210"
 export GPC_CONSUMER_GPC_STRUCTURED_PATH="/GP0001/STU3/1/gpconnect/fhir/Patient/\$gpc.getstructuredrecord"
 export GPC_CONSUMER_GPC_GET_DOCUMENT_PATH="/GP0001/STU3/1/gpconnect/fhir/Binary/{documentId}"
 export GPC_CONSUMER_GPC_GET_PATIENT_PATH="/GP0001/STU3/1/gpconnect/fhir/Patient"
+export GPC_CONSUMER_URL="http://localhost:8080"
+export GPC_CONSUMER_GPC_GET_PATIENT_DOCUMENT_PATH="/GP0001/STU3/1/gpconnect/fhir/Patient/**"
```

Comment:
I'd suggest renaming this to something like `GPC_CONSUMER_SEARCH_DOCUMENTS_PATH` to avoid confusion with GET_DOCUMENT